workmail-update-email/src/utils.py

```diff
@@ -192,8 +193,9 @@ def update_email(downloaded_email, email_subject, flow_direction):
     """
     updated_email = update_email_body(downloaded_email)
     # Only update subject of an incoming email
-    if flow_direction == 'INBOUND':
-        new_subject =  f"[EXTERNAL EMAIL] {email_subject}"
+    if flow_direction == 'INBOUND' and subject_tag:
+        new_subject =  f"{subject_tag}{email_subject}"
+        logger.info("New subject: " + new_subject)
```

Comment:
Yes, there is no reason to log the modified subject since the modification itself is constant, so the log really only needs to convey that the modification happened